web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.jsx

```diff
@@ -458,7 +462,7 @@ const MemberSelectorDialog = ({
             disabled={checked.size === 0}
             onClick={handleSubmit}
           >
-            Add
+            Save
```

Comment:
We are no longer just adding members to the list of selected members. We can also remove them. So labeling this "Save" makes more sense.